src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -69,16 +71,22 @@
     private GenericContainer<?> cntn;
 
     /**
-     * Artipie server port.
+     * URL 'http://host:port/my-nuget'.
      */
-    private int port;
+    private String url;
+
+    /**
+     * Package name in the temporary directory.
+     */
+    private String pckg;
 
     @BeforeEach
     void init() throws Exception {
         final String name = "my-nuget";
         this.server = new ArtipieServer(this.tmp, name, this.config());
-        this.port = this.server.start();
-        Testcontainers.exposeHostPorts(this.port);
+        final int port = this.server.start();
+        this.url = String.format("http://host.testcontainers.internal:%d/my-nuget", port);
```

Comment:
@genryxy `this.url` field is initialized only after `this.config()` method call. But it is used inside `config()` method. Not sure how this test works.

Reply:
@olegmoz it works normal since I'm recording file with config after the port number is known